.jenkins/pytorch/test.sh

```diff
@@ -14,6 +14,8 @@ echo "Testing pytorch"
 if [ -n "${IN_CIRCLECI}" ]; then
   # TODO move this to docker
   pip_install unittest-xml-reporting coverage
+  sudo apt-get -qq update
+  sudo apt-get -qq install lcov
```

Comment:
Is it possible to just add this to the base dockerfile instead of installing it in this test script?

Reply:
Yes, this is better. Do you know where is the base dockerfile config?

Reply:
Besides, I also wonder why we don't put it under `if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then...` at line 36